salt/metalk8s/kubeadm/init/control-plane/lib.sls

```diff
@@ -0,0 +1,7 @@
+# Share macro for building image names for each control-plane component
+{% set registry_prefix = "localhost:5000/metalk8s-2.0/" %}
+{% set kubernetes_version = "1.11.7" %}
+
+{% macro get_image_name(component) -%}
+"{{ registry_prefix }}/{{ component }}:{{ kubernetes_version }}"
```

Comment:
This doesn't work. It generates a value which has 2 `/`s, because `registry_prefix` already has a `/` at the end. Subsequently, the image name is in an invalid format, and pulling fails.

Suggestion:

- Rename `registry_prefix` to `repository`, and set its value to `localhost:5000/metalk8s-2.0`
- In the macro, use `repository` and keep the `/` as-is.

Reply:
Indeed I did see that one and forgot to clean it up, thanks!